velox/functions/sparksql/RegexFunctions.cpp

```diff
@@ -82,4 +82,18 @@ std::shared_ptr<exec::VectorFunction> makeRegexExtract(
   return result;
 }
 
+std::shared_ptr<exec::VectorFunction> makeRegexReplace(
+    const std::string& name,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& config) {
+  auto result = makeRe2Replace(name, inputArgs, config);
+  // The below line is commented out and handled within the RegexReplace
+  // functions. This enables input of Vectors of patterns
+  // instead of just constant patterns.
+  // This was done in order to emulate specific unit tests done
+  // in Spark SQL.
+  // ensureRegexIsCompatible("REGEXP_REPLACE", inputArgs[1].constantValue);
```

Comment:
consider removing comment here, and place it where you are handling it.

Reply:
Comment is there to explain why the function is missing compared to its neighbors. The implementation has its own comments to explain